frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/obc-test.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { expect } from 'chai';
+import {
+  ACCESS_KEY,
+  BOUND,
+  MASK,
+  MINUTE,
+  NS,
+  OBC_NAME,
+  OBC_STORAGE_CLASS,
+  SECOND,
+  SECRET_KEY,
+} from '../utils/consts';
+import { testDeployment, deployment } from '../mocks/deploymentData';
+import { CreateOBCHandler } from '../views/obcPage';
+import { testName } from '../../../integration-tests-cypress/support/index';
+
+describe('Test Object Bucket Claim resource', () => {
+  let obcHandler;
+  let obcUrl;
+
+  before(() => {
+    cy.url().then((url) => {
+      if (url.match(/oauth/)) {
+        cy.login();
+      }
+    });
+    cy.visit('/');
+    cy.install();
+    obcHandler = new CreateOBCHandler(OBC_NAME, testName, OBC_STORAGE_CLASS, '');
+    obcHandler.createBucketClaim();
+    cy.exec(`echo '${JSON.stringify(deployment)}' | kubectl create -n ${testName} -f -`);
+    cy.url().then((url) => {
+      obcUrl = url;
+    });
+  });
+
+  beforeEach(() => {
+    cy.wait(10 * SECOND);
+  });
+
+  it('Test if Object Bucket Claim is created sucessfully', () => {
+    cy.visit(obcUrl);
+    cy.get('.co-resource-item__resource-name').should('be.visible');
+    cy.byLegacyTestID('resource-title')
+      .invoke('text')
+      .should('eq', OBC_NAME);
+    cy.get('.co-resource-item__resource-status', { timeout: 1 * MINUTE })
+      .should('be.visible')
+      .invoke('text')
+      .should('eq', BOUND);
+  });
+
+  it('Test if owner and creation date are shown correctly', () => {
```

Comment:
Does this really need to be a different `it` block? Every `it` block should be able to execute independently.

Reply:
I tried to stick as close to the pattern defined in the `obcTest.scenario` file (the one we migrated from) as possible, for better readability between the two in the future. It defines a similar `it` block as you can see below.

https://github.com/openshift/console/blob/8524169bea23554b6e54c669dea9f6d55ec1fa7e/frontend/packages/noobaa-storage-plugin/integration-tests/tests/obcTest.scenario.ts#L36

Reply:
Also, just to add to this, this `it` block should be able to execute independently, as it checks for information that's independent  of the other `it` blocks.

Reply:
Doesn't it depend upon https://github.com/openshift/console/pull/8061/files#diff-bc97b04fc2ffc9319572ed03433dc859d1ffacac875f13eb19c0df1874650ffaR42 
Which is getting executed in the above test block.
The goal of this migration is to write tests in Cypressy manner and not only port code. All our tests should adhere to Cypress best practices. PTAL https://docs.cypress.io/guides/references/best-practices.html

Reply:
I have removed the `cy.visit` part from the test above, it seems it was redundant at this point as the before block already takes us to the required page. I commented out every `it` other than the one we're talking about here and it seems work independently.

![image](https://user-images.githubusercontent.com/33557095/107334730-17876d00-6add-11eb-8dfe-77d552da6ec0.png)

I'm refactoring the test structure based on the best practice link you quoted. Sending a patch soon.